pkg/server/container_stats_list.go

```diff
@@ -107,10 +107,12 @@ func (c *criService) getContainerMetrics(
 				UsageCoreNanoSeconds: &runtime.UInt64Value{Value: metrics.CPU.Usage.Total},
 			}
 		}
-		if metrics.Memory != nil && metrics.Memory.Usage != nil {
+		if metrics.Memory != nil {
```

Comment:
since we dont check `metrics.Memory.Usage != nil` anymore, missing memory metrics result in a adding a value with WorkingSet = 0, instead of `cs.Memory = nil`.  I don't know the codebase, so I don't know if this matters.

Reply:
Hm, good catch. I'll keep the behavior unchanged.